bloom-housing/bloom
# docs: update readmes

## Description

Updates each of our READMEs, at first just to add better setup instructions as I setup the new laptop, but I noticed we had quite a bit of outdated information across all of them and some incorrectly configured environment variable template defaults for starting from fresh.

## How Can This Be Tested/Reviewed?

The diffs are kinda gnarly, you can also `View File` on each file to see a markdown preview (the links below)
* [Root](https://github.com/bloom-housing/bloom/blob/fed48ab1beb34f66ac443f3a4b48c6eb95de6d80/README.md)
* [Api](https://github.com/bloom-housing/bloom/blob/fed48ab1beb34f66ac443f3a4b48c6eb95de6d80/api/README.md)
* [Public](https://github.com/bloom-housing/bloom/blob/fed48ab1beb34f66ac443f3a4b48c6eb95de6d80/sites/public/README.md)
* [Partners](https://github.com/bloom-housing/bloom/blob/fed48ab1beb34f66ac443f3a4b48c6eb95de6d80/sites/partners/README.md)
* [Shared helpers](https://github.com/bloom-housing/bloom/blob/fed48ab1beb34f66ac443f3a4b48c6eb95de6d80/shared-helpers/README.md)

## Author Checklist:


## Review Process:

- Read and understand the issue
- Ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Either (1) explicitly ask a clarifying question, (2) request changes, or (3) approve the PR, even if there are very small remaining changes, if you don't need to re-review after the updates
